devops/openshift/load-test-gateway-deploy.yml

```diff
@@ -0,0 +1,196 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: ${NAME}
+labels:
+  project: ${PROJECT}
+  service: ${SERVICE_NAME}
+objects:
+  - apiVersion: apps.openshift.io/v1
+    kind: DeploymentConfig
+    metadata:
+      name: ${NAME}
+    spec:
+      replicas: "${{REPLICAS}}"
+      revisionHistoryLimit: 10
+      selector:
+        deploymentconfig: ${NAME}
+      strategy:
+        type: Rolling
+      template:
+        metadata:
+          labels:
+            deploymentconfig: ${NAME}
+        spec:
+          containers:
+            - env:
+                - name: VERSION
+                  value: ${BUILD_TAG}
+                - name: NODE_ENV
+                  value: ${NODE_ENV}
+                - name: LOAD_TEST_API_PORT
+                  value: "${PORT}"
+                - name: KEYCLOAK_REALM
+                  value: ${KEYCLOAK_REALM}
+                - name: KEYCLOAK_AUTH_URL
+                  value: ${KEYCLOAK_AUTH_URL}
+                - name: POSTGRES_HOST
+                  value: "${DB_SERVICE}"
+                - name: POSTGRES_USER
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DB_USERNAME_KEY}
+                      name: ${DB_SECRET_NAME}
+                - name: POSTGRES_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DB_PASSWORD_KEY}
+                      name: ${DB_SECRET_NAME}
+                - name: POSTGRES_DB
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${SIMS_DB_NAME}
+                      name: ${LOAD_TEST_GATEWAY_SECRET_NAME}
+                - name: DISABLE_ORM_CACHE
```

Comment:
Is `DISABLE_ORM_CACHE` needed for the `load-test-gateway`?

Reply:
Not necessarily, but I felt it is better to ignore the orm cache part if we are not providing redis context to the application